public/app/features/plugins/extensions/registry.ts

```diff
@@ -38,14 +38,14 @@ export function createPluginExtensionsRegistry(apps: Record<string, AppPluginCon
 }
 
 function createRegistryItem(pluginId: string, extension: PluginsExtensionLinkConfig): PluginsExtensionLink {
-  const href = `/a/${pluginId}${extension.path}`;
+  const path = `/a/${pluginId}${extension.path}`;
 
   return Object.freeze({
     type: PluginExtensionTypes.link,
     title: extension.title,
     description: extension.description,
-    href: href,
-    key: hashKey(`${extension.title}${href}`),
+    path: path,
```

Comment:
Super super nit: maybe we can either move the variable declaration to this line or just use the assignment shorthand?
```suggestion
    path: `/a/${pluginId}${extension.path}`,
```

Reply:
hmm, the path is being used in the hashKey line as well.

Reply:
Ohh, I missed that, sorry!